Samples/YOLOv4ObjectDetection/YOLOv4ObjectDetection/TinyYOLO.cs

```diff
@@ -0,0 +1,47 @@
+// This file was automatically generated by VS extension Windows Machine Learning Code Generator v3

```

Comment:
are you actually using this file or did it just come in from MLGen? We should remove it if it's not being used.